input/fsh/ValueSet-DiseaseCodes.fsh

```diff
@@ -5,4 +5,4 @@ Description: "This value set includes all codes from NCIt (NCI Thesaurus)."
 * ^version = "0.1.0"
 * ^status = #draft
 * ^experimental = false
-* include codes from system $ncit.owl
```

Comment:
Are we deprecating NCIt? We actually would want to include NCIt, OMIM, and MONDO since they are pretty standard disease ontologies in biomed.

Reply:
And please update the `Description` above!

Reply:
As for Mondo and NCIt, I don't have those CodeSystems/ValueSets at this time, and needed these changes for a an updated and functional model for some tests. I feel like this may make a good choice for an issue if someone needs them sooner, or, perhaps if we end up with a terminology server with all of our ontologies loaded, we could add them to the valueset then.

Reply:
Let's modify the current disease example resource so that it would use a code from OMIM.

Reply:
Added some example codes for Mondo and NCIt and updated the disease instance to use one from that set.